### ISSUE TITLE
Query parameter to set utteranceQueue enabled

### ISSUE THREAD

**Issue by zepumph** [phetsims/scenery-phet#380]
From the a11y sprint, especially in regards to Area Model work with @jonathanolson dealing with `AccessibleNumberTweaker`, I think we want a want to turn off partially implemented dynamic/interactive descriptions and alerts. One good thing would be to turn off utterance queue.

@jessegreenberg what do you think? We could wrap the `utteranceQueue.initialize();` in `Sim.js` in a query parameter conditional.

Or maybe just disable it after initialization. We also, similar to the `accessibility` flag in general, want a way to pass that in to the `Sim.js` constructor..

**Comment by zepumph** [comment]
tagging @samreid because he often has good opinions.

**Comment by zepumph** [comment]
This may be an insufficient, or "half measure" issue. Perhaps we should develop a system for toggling all desired a11y features both by query parameter, and in the sim constructor. So far we have 3 "levels." 

(a) keyboard navivation
(b) static descriptions (the first two may be combined into the same level)
(c) dynamic descriptions and interactive alerts (mostly controlled by the utteranceQueue, put perhaps there is more)

**Comment by samreid** [comment]
I'd love to hear more about the motivation for this.  Is the main reason because we will want to publish sims that have (a) and (b) but not (c)?  Or is it more that it an incomplete (c) disrupts the development process?  Or something else?

Or is it more that we would want to disable the utterance queue for components that aren't "done" yet, but leaving it on for the rest of the sim?

**Comment by zepumph** [comment]
I think of it as (a) and (b) but not (c). There is a push from the a11y side of sim development to be able to publish simulations with different levels of accessibility implemented. The design time to add interactive alerts to a simulation is often not there, so it is important to be able to have keyboard nav and static descriptions complete and isolated so that they can be published on their own.

If a common code component has interactive alerts built in, then we will either need to design the descriptions for those, or turn them off. Business as usual will likely sound be broken and confusing for screen readers.

**Comment by pixelzoom** [comment]
When we last left this issue (May 2018!) @zepumph asked @jessegreenberg if this issue was worth pursuing.  @jessegreenberg is this worth pursing, or can we close this issue?

**Comment by zepumph** [comment]
I do not think this is the direction we should or are going. In general my type-a personality still believes that features should have an off switch so that pieces of description don't leak into a sim without it fully outfitted. But in general that is not the philosophy that PhET, or the web-accessibility community at large, as adhered to. We will instead devote resources to outfitting the feature in sims.

@jessegreenberg please close or add your thoughts.

**Comment by jessegreenberg** [comment]
I am fine closing this, if this is useful for development or requested by the design team someday we can add it. But it hasn't really come up since this issue was opened.
